app/assets/javascripts/app/form-validation.js

```diff
@@ -2,7 +2,6 @@ import h5f from 'h5f';
 
 import validateField from './validate-field';
 
-
```

Comment:
OK, updated at 4fc7230 -- I looked at some other js files and saw that we usually do not do a newline *Between* imports so changed that here, too